src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryConnectionProperties.java

```diff
@@ -16,11 +16,10 @@
 
 package org.springframework.cloud.deployer.spi.cloudfoundry;
 
-import java.net.URL;
+import org.springframework.validation.annotation.Validated;
```

Comment:
Reshuffled import order. Please use the original order.

Reply:
revered to the original order